src/components/commons/Avatar/index.tsx

```diff
@@ -27,7 +27,7 @@ export const Avatar = ({ size, profileImageUrl, isActivated }: AvatarProps) => {
         {profileImageUrl ? (
           <Image src={profileImageUrl} alt='profileImage' layout='fill' />
         ) : (
-          <Image src={SVGS.profile.default.url} alt={SVGS.profile.default.alt} width={imageSize} height={imageSize} />
+          <Image src={SVGS.profile.url} alt={SVGS.profile.alt} width={imageSize} height={imageSize} />
```

Comment:
SVGS에서 url 가져오실 때 아래처럼 디스트럭쳐링 하고 사용하시면 좋을 것 같습니다!

```suggestion
const { profile: { url, alt } } = SVGS;

// ...

          <Image src={url} alt={alt} width={imageSize} height={imageSize} />
```